store-redis/src/test/groovy/com/mx/redis/RedisStoreTest.groovy

```diff
@@ -169,4 +172,49 @@ class RedisStoreTest extends Specification implements WithMockery {
     then:
     subject.status() == "OK"
   }
+
+  def "rethrows connection failure"() {
+    given:
+    subject.setConnection(null)
+    subject = spy(subject)
```

Comment:
I usually add `spy` when instantiating the `subject`, so it doesn't need to be added to individual tests.

```
// in setup()
subject = spy(new RedisStore(configurations))
```

Reply:
Fixed